projects/observability/src/pages/custom-dashboards/custom-dashboard-detail/custom-dashboard-detail.component.ts

```diff
@@ -68,6 +75,7 @@ export class CustomDashboardDetailComponent {
     this.activedRoute.params.subscribe(params => {
       this.dashboardId = params.dashboard_id;
       this.isNew = params.dashboard_id === 'create';
+      this.dashboardView = params.dashboard_view;
```

Comment:
Do you rather want to save a derived variable here? Something like below, given we are doing the `if` check 2-3 times in the component.

```javascript
this.isMyDashboardTab = params.dashboard_view === DASHBOARD_VIEWS.MY_DASHBOARDS;
```

That can help remove `if` condition from everywhere in the page.